CHANGELOG.md

```diff
@@ -4,6 +4,16 @@
 
 ### 🛑 Breaking changes 🛑
 
+### 🚩 Deprecations 🚩
+
+### 💡 Enhancements 💡
+
+### 🧰 Bug fixes 🧰
+
+## v0.52.0 Beta
+
+### 🛑 Breaking changes 🛑
+
 - Remove `configunmarshaler.Unmarshaler` interface, per deprecation comment (#5348)
```

Comment:
Cannot comment on that line, but on line 23 (in the new file) there is a typo, `derecated` -> `deprecated`

Reply:
Fixed, thanks for catching